api/v1/client/client.go

```diff
@@ -99,7 +98,8 @@ func (c *Client) Call(ctx context.Context, method string, rel *url.URL, body, ou
 		case string:
 			bodyData = []byte(val)
 		default:
-			bodyData, err = jsonapi.Marshal(body)
+			// TODO: not sure if it safe, should we introduce another Call ?
```

Comment:
Sure! Sorry, my bad, I wasn't clear enough, and it seems that I should bring this into the description since it's one of my concerns about this PR.

Let me fix it quickly!